src/sagas/RootSaga.js

```diff
@@ -70,7 +68,7 @@ export function* handleFetchProfile(action) {
     if (e.response.status === 404) {
       yield put(push('/notfound'));
     } else {
-      yield put(fetchProfileFailure(e.message));
+      yield put(push('/error'));
```

Comment:
1. Do we want `LoggingService.logAPIErrorResponse(e)` here?
2. Should `fetchProfileFailure` still exist if it isn't used?

Reply:
1. We do have it above ^^.
2. No, I will remove it.